Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.yml

```diff
@@ -0,0 +1,2082 @@
+category: Endpoint
+commonfields:
+  id: VMware Carbon Black EDR
```

Comment:
if `VMware Carbon Black EDR v2` is available please change the name and id to it as well.